php_phongo_structs.h

```diff
@@ -204,25 +204,25 @@ typedef struct {
 	PHONGO_ZEND_OBJECT_PRE
 	mongoc_client_t    *client;
 	char               *command_name;
-	uint32_t            server_id;
+	int                 server_id;
```

Comment:
This is curious. For some of the event_ts we use int, and sometimes uint32_t for this. One of them is right...